src/util/beacon.ts

```diff
@@ -26,7 +28,11 @@ const hasBeaconSupport = "sendBeacon" in navigator
 export default function beacon(
     url: string,
     data: string,
+    method: BeaconMethod,
 ): Promise<SendBeaconResult> {
+    if (method == "GET") {
```

Comment:
If we used an enum we could use the token here instead of the string:
```suggestion
    if (method == BeaconMethod.Get) {
```

Reply:
I suspect this makes life easier for the compiler to optimize file size. 👍